### ISSUE TITLE
[BUG]为什么我跑出来全是哀叹的声音，demo跑不动

### ISSUE THREAD

**Issue by Kydon-ai** [MoonshotAI/Kimi-Audio#138]
[output1.zip](https://github.com/user-attachments/files/21525227/output1.zip)

<img width="1106" height="633" alt="Image" src="https://github.com/user-attachments/assets/579ce186-b7eb-4d5f-a72b-123e28fbdaf8" />
上面的zip是Kimi-Audio-7B跑出来的
下面的报错是Kimi-Audio-7B-Instruct跑出来的
有没有遇到一样问题的同学？

**Comment by Kydon-ai** [comment]
accelerate==1.9.0
aiohappyeyeballs==2.6.1
aiohttp==3.12.15
aiosignal==1.4.0
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
asttokens==3.0.0
async-timeout==5.0.1
attrs==25.3.0
audioread==3.0.1
beautifulsoup4==4.13.4
blobfile==3.0.0
cairocffi==1.7.1
CairoSVG==2.8.2
certifi==2025.8.3
cffi==1.17.1
charset-normalizer==3.4.2
colorama==0.4.6
comm==0.2.3
conformer==0.3.2
contourpy==1.3.2
cssselect2==0.8.0
cycler==0.12.1
datasets==4.0.0
debugpy==1.8.16
decorator==5.2.1
decord==0.6.0
deepspeed==0.16.9
defusedxml==0.7.1
diffusers==0.34.0
dill==0.3.8
easydict==1.13
einops==0.8.1
exceptiongroup==1.3.0
executing==2.2.0
filelock==3.18.0
fire==0.7.0
fonttools==4.59.0
frozenlist==1.7.0
fsspec==2025.3.0
gdown==5.2.0
hf-xet==1.1.7
hjson==3.1.0
huggingface-hub==0.34.3
HyperPyYAML==1.2.2
idna==3.10
immutabledict==4.2.1
importlib_metadata==8.7.0
ipykernel==6.30.1
ipython==8.37.0
ipywidgets==8.1.7
jedi==0.19.2
Jinja2==3.1.6
joblib==1.5.1
jsonlines==4.0.0
jupyter_client==8.6.3
jupyter_core==5.8.1
jupyterlab_widgets==3.0.15
kiwisolver==1.4.8
lazy_loader==0.4
librosa==0.11.0
lightning-utilities==0.15.2
llvmlite==0.44.0
loguru==0.7.3
lxml==6.0.0
markdown-it-py==3.0.0
MarkupSafe==3.0.2
matplotlib==3.10.5
matplotlib-inline==0.1.7
mdurl==0.1.2
modelscope==1.28.2
more-itertools==10.7.0
mpmath==1.3.0
msgpack==1.1.1
multidict==6.6.3
multiprocess==0.70.16
nest-asyncio==1.6.0
networkx==3.4.2
ninja==1.11.1.4
numba==0.61.2
numpy==2.2.6
nvidia-cublas-cu12==12.1.3.1
nvidia-cuda-cupti-cu12==12.1.105
nvidia-cuda-nvrtc-cu12==12.1.105
nvidia-cuda-runtime-cu12==12.1.105
nvidia-cudnn-cu12==9.1.0.70
nvidia-cufft-cu12==11.0.2.54
nvidia-curand-cu12==10.3.2.106
nvidia-cusolver-cu12==11.4.5.107
nvidia-cusparse-cu12==12.1.0.106
nvidia-cusparselt-cu12==0.6.2
nvidia-ml-py==13.580.65
nvidia-nccl-cu12==2.20.5
nvidia-nvjitlink-cu12==12.4.127
nvidia-nvtx-cu12==12.1.105
omegaconf==2.3.0
openai-whisper==20250625
packaging==25.0
pandas==2.3.1
parso==0.8.4
pastel==0.2.1
pexpect==4.9.0
pillow==11.3.0
platformdirs==4.3.8
poethepoet==0.10.0
pooch==1.8.2
portalocker==2.0.0
prompt_toolkit==3.0.51
propcache==0.3.2
psutil==7.0.0
ptyprocess==0.7.0
pure_eval==0.2.3
py-cpuinfo==9.0.0
pyarrow==21.0.0
pycparser==2.22
pycryptodomex==3.23.0
pydantic==2.11.7
pydantic_core==2.33.2
Pygments==2.19.2
pyparsing==3.2.3
PySocks==1.7.1
python-dateutil==2.9.0.post0
pytorch-lightning==2.5.2
pytz==2025.2
PyYAML==6.0.2
pyzmq==27.0.1
regex==2025.7.34
requests==2.32.4
rich==14.1.0
ruamel.yaml==0.18.14
ruamel.yaml.clib==0.2.12
sacrebleu==1.5.1
safetensors==0.6.1
scikit-learn==1.7.1
scipy==1.15.3
sentencepiece==0.2.0
six==1.16.0
soundfile==0.13.1
soupsieve==2.7
sox==1.5.0
soxr==0.5.0.post1
stack-data==0.6.3
sty==1.0.6
sympy==1.13.1
termcolor==3.1.0
threadpoolctl==3.6.0
tiktoken==0.10.0
timm==1.0.19
tinycss2==1.4.0
tokenizers==0.21.4
tomlkit==0.13.3
torch==2.4.1
torchaudio==2.4.1
torchcde==0.2.5
torchdiffeq==0.2.5
torchdyn==1.0.6
torchmetrics==1.8.0
torchsde==0.2.6
torchvision==0.19.1
tornado==6.5.1
tqdm==4.67.1
traitlets==5.14.3
trampoline==0.1.2
transformers==4.55.0
triton==3.0.0
typing-inspection==0.4.1
typing_extensions==4.14.1
tzdata==2025.2
ujson==5.10.0
urllib3==2.5.0
validators==0.35.0
wcwidth==0.2.13
webencodings==0.5.1
wget==3.2
widgetsnbextension==4.0.14
xxhash==3.5.0
yarl==1.20.1
zipp==3.23.0

**Comment by Kydon-ai** [comment]
我换了个环境是可以运行的，上面是pip freeze导出的requirements.txt
额外的，我使用
flash-attn @ file:///XXXXXXXX/Kimi-Audio/github.com/Dao-AILab/flash-attention/releases/download/v2.7.3/flash_attn-2.7.3%2Bcu12torch2.4cxx11abiFALSE-cp310-cp310-linux_x86_64.whl#sha256=15aae40ce1b09f613ea943dd55f425c57e7496661b113fada685520d9339aea3
可以自行下载
参考：https://blog.csdn.net/ha_xianren/article/details/148007173
